src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+
+import {
+  DEFAULT_PROPS,
+  STATS_ICONS,
+  STATS_LABELS,
+  DEFAULT_AVATAR_ICON,
+  FALLBACK_INITIALS,
+} from './ProfileCard.constants';
+import {
+  StyledCard,
+  StyledProfileContainer,
+  StyledAvatar,
+  StyledInfoContainer,
+  StyledName,
+  StyledUsername,
+  StyledRole,
+  StyledStatsContainer,
+  StyledStatItem,
+  StyledStatIcon,
+  StyledStatContent,
+  StyledStatValue,
+  StyledStatLabel,
+} from './ProfileCard.styles';
+
+import type { ProfileCardProps, StatsItemProps } from './ProfileCard.types';
+
+const StatsItem: React.FC<StatsItemProps> = ({ icon, label, value, 'data-testid': dataTestId }) => {
+  const formattedValue = typeof value === 'number' ? value.toLocaleString('en-US') : value;
+
+  return (
+    <StyledStatItem $variant={DEFAULT_PROPS.variant} data-testid={dataTestId}>
+      <StyledStatIcon>{icon}</StyledStatIcon>
+      <StyledStatContent $variant={DEFAULT_PROPS.variant}>
```

Comment:
Same issue: StatsItem's StyledStatContent is passed `DEFAULT_PROPS.variant`. Because both StyledStatItem and StyledStatContent receive the default rather than the current `variant`, the expanded/compact layout won't be applied to stats.

**Sugerencia:**
```typescript
See previous suggestion: accept and forward `$variant` from parent into StatsItem and pass it to both StyledStatItem and StyledStatContent.
```